### ISSUE TITLE
Negative number as argument throws error "Invalid option"

### ISSUE THREAD

**Issue by jsfi** [75lb/command-line-args#10]
If I try to parse a flag with a negative value like --one="-1" the parser throws the following error:
```
[Error: Invalid option: -1]
```

You can test this by extending the type-number.js tests with:
```
t.deepEqual(
  cliArgs(optionDefinitions).parse([ '--one', '-1' ]),
  { one: -1 }
)
````

**Comment by 75lb** [comment]
well spotted, thanks.. looking into it.

**Comment by 75lb** [comment]
released in v2.0.0 - thanks for the report
